examples/relay-chat/README.md

```diff
@@ -0,0 +1,23 @@
+# Using Waku Relay WebRTC
+
+**Demonstrates**:
+
+- Waku Relay node with direct WebRTC connection 
+- Pure Javascript/HTML.
+
+This example uses Waku Filter to listen to messages and Waku Light Push to send messages.
+
+To test the example run `npm install` and then `npm start`.
+
+The `master` branch's HEAD is deployed at https://examples.waku.org/relay-chat/.
+
+### Steps to run an example: 
+1. Build and then run `go-waku` node with following command: `./build/waku --ws true --relay true --circuit-relay true`
```

Comment:
@richard-ramos do we have some go-waku nodes running somewhere we could use?

Also, is the issue that circuit-relay is not supported in nwaku? cc @waku-org/nwaku-developers

Reply:
There is go-waku.prod and go-waku.test fleets (https://fleets.status.im/), they're currently connected to status.prod and status.test respectively so expect heavy bandwidth usage with relay

Reply:
as I see only `go-waku` has `circuit-relay` implemented
> Also, is the issue that circuit-relay is not supported in nwaku?

Reply:
after experimenting I see that it is possible to use `nwaku` nodes for that use-case but it happens often that a `NO_RESERVATION` error is thrown and the connection fails

I didn't investigate it thoroughly and it might be possible to improve or mitigate it but for the example it is not strictly necessary 

given that leaving it with a hardcoded value for now and will follow up in the next iterations if needed